docs/Permissions/Permissioning.md

```diff
@@ -0,0 +1,131 @@
+<!--- END of page meta data -->
+
+# Permissions 
+
+A permissioned network is a network where only specified nodes and accounts (participants) can participate. 
+Nodes and accounts outside those specified are prevented from participating. 
+
+## Node Whitelisting 
+
+Node whitelisting is specified by the nodes whitelist in the [`permissions_config.toml`](#permissions-configuration-file) file. 
+A node with node whitelisting enabled communicates only with nodes in the nodes whitelist. 
+
+!!! example "Nodes Whitelist in `permissons_config.toml`"
+    `nodes-whitelist=["enode://6f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0@192.168.0.9:4567","enode://6f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0@192.169.0.9:4568"]`
+
+Node whitelisting is at the node level. That is, each node in the network has a [`permissions_config.toml`](#permissions-configuration-file)
+file in the [data directory](../Reference/Pantheon-CLI-Syntax.md#data-path) for the node.  
+
+To add or remove nodes from the nodes whitelist when the node is running, use the JSON-RPC API methods:
+ 
+* [perm_addNodesToWhitelist](../Reference/JSON-RPC-API-Methods.md#perm__addnodestowhitelist)
+* [perm_removeNodesFromWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_removeNodesFromWhiteList)
+* [perm_getNodesWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_getNodesWhiteList)
+
+!!! note
+    Each node has a `permissions_config.toml` file which means nodes in the network can have different node whitelists. 
+    This means nodes may be participating in the network that are not on the whitelist of other nodes in the network. 
+    We recommend each node in the network has the same nodes whitelist. 
+    
+    On-chain permissioning is under development. On-chain permissioning will use one on-chain 
+    node whitelist. 
+    
+!!! example "Example Different Node Whitelists" 
+
+    Node 1 Whitelist = [Node 2, Node 3] 
+    
+    Node 2 Whitelist = [Node 3, Node 5] 
+    
+    Node 5 is participating in the same network as Node 1 even though Node 1 does not have Node 5 on their whitelist.
+
+### Bootnodes
+
+The bootnodes must be included in the nodes whitelist. 
+
+!!! example 
+    If you start Pantheon with:
+    
+     ```bash
+     --bootnodes="enode://7e4ef30e9ec683f26ad76ffca5b5148fa7a6575f4cfad4eb0f52f9c3d8335f4a9b6f9e66fcc73ef95ed7a2a52784d4f372e7750ac8ae0b544309a5b391a23dd7@127.0.0.1:30303","enode://2feb33b3c6c4a8f77d84a5ce44954e83e5f163e7a65f7f7a7fec499ceb0ddd76a46ef635408c513d64c076470eac86b7f2c8ae4fcd112cb28ce82c0d64ec2c94@127.0.0.1:30304","enode://7b61d5ee4b44335873e6912cb5dd3e3877c860ba21417c9b9ef1f7e500a82213737d4b269046d0669fb2299a234ca03443f25fe5f706b693b3669e5c92478ade@127.0.0.1:30305" 
+     ```
+    
+    The `nodes-whitelist` in `permissioning_config.toml` must contain the specified bootnodes. 
```

Comment:
do you want to say what happens if they aren't?

Reply:
Added